lib/ui/geometry.dart

```diff
@@ -1749,6 +1752,17 @@ class RRect {
 ///
 /// Used by [Canvas.drawAtlas]. This is a more efficient way to represent these
 /// simple transformations than a full matrix.
+///
+/// It is equivalent to a [Matrix4] built using the following parameters (where
+/// `rst` is the corresponding `RSTransform` instance):
+///
+/// ```dart
+/// // Matrix4's constructor arguments are in column-major order.
+/// Matrix4(rst.scos, rst.ssin, 0, 0,
```

Comment:
snippet analysis is sad that `rst` isn't initialized.

Reply:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8748370038705622465/+/u/run_test.dart_for_analyze_shard_and_subshard_None/stdout